dask/dataframe/io/parquet/core.py

```diff
@@ -185,7 +186,7 @@ def read_parquet(
     index=None,
     storage_options=None,
     engine="auto",
-    use_nullable_dtypes=False,
+    use_nullable_dtypes: bool | Literal["pandas", "pyarrow"] = False,
```

Comment:
Btw, @mroeschke this is the PR I mentioned offline about extending `use_nullable_dtypes` to support `"pandas"` and `"pyarrow"`

Reply:
Ah yeah this is pretty clean!

Our pandas issues https://github.com/pandas-dev/pandas/issues/48957 (offline discussion happened here) and https://github.com/pandas-dev/pandas/issues/49997 are examples where some discussion/preference of keeping `use_nullable_dtypes` boolean